src/debops/_data/defaults.toml

```diff
@@ -1,7 +1,3 @@
-{# Copyright (C) 2020-2021 Maciej Delmanowski <drybjed@gmail.com>
```

Comment:
Yet another reason not to add yet another file format (TOML) when we already widely use YAML. Anyway, I suggest you use something like this:

```
#{# Copyright (C) 2020-2021 Maciej Delmanowski <drybjed@gmail.com>
```

instead because the reuse.software spec recommends to include the copyright in the file when possible.

Reply:
The template file is used only internally in the script to generate defaults in the runtime configuration structure. The Jinja `{# #}` are not needed here, a mistake on my part during development. I'll fix that in a commit before merging this PR.